### PR TITLE
Exp sharing updates

### PR DESCRIPTION
**Pull request by dberenbaum**
Includes Studio sharing and reorganizes info

### CODE REVIEW

**Comment by shcheklein** [comment]
@dberenbaum there are some failing links + you forgot to include the image it looks like (it fails to deploy bc of that):

```
error Image Not Found: Image "/img/exp-sharing-studio.png" not found in folder "static" referenced from "/tmp/gatsby-build/content/docs/user-guide/experiment-management/sharing-experiments.md". Please check static folder name and that file exists at "static/img/exp-sharing-studio.png".
```

otherwise looks good to me !

**Comment by tapadipti** [comment]
@dberenbaum I've created [another PR on top of this](https://github.com/iterative/dvc.org/pull/4487) to complete the Studio docs updates for exp push. Pls review.

File: content/docs/user-guide/experiment-management/sharing-experiments.md [hunk 1]
```diff
@@ -32,56 +23,62 @@ Git and [DVC remotes][remote storage] as needed:
...
 
-For problems accessing the Git remote, see the [remote storage] guide.
+View and manage pushed experiments in [Studio](https://studio.iterative.ai). To
+notify Studio when you push experiments,
+[get your Studio token](https://studio.iterative.ai/user/_/profile?section=accessToken)
+and save it with
+[`dvc config --global studio.token ***`](/doc/user-guide/project-structure/configuration#studio).
 
-For problems accessing the DVC remote, see [troubleshooting].
+![Sharing experiments in Studio](/img/exp-sharing-studio.png)
```
Comment (tapadipti) [inline content/docs/user-guide/experiment-management/sharing-experiments.md:46]: This image is missing.

**Comment by yathomasi** [comment]
It looks like the missing image is causing the Heroku failure. Closing the issue to stop the Heroku Deployment Hell. Feel free to reopen with the fix.
Screenshot 2023-04-24 at 12 49 09

**Pull request closed without merging**